src/cli.js

```diff
@@ -8,6 +8,9 @@ import {Program} from './program';
 
 export function main() {
   let program = new Program();
+  // yargs uses magic camel case expansion to expose options on the
+  // final argv object. For example, the 'build-dir' option is available
+  // as argv.buildDir.
```

Comment:
I see this as more that it always provides a camelCased alias. iirc if you have a foo-bar option it is still available as `foo-bar` as well as `fooBar`.